DelvUI/Interface/GeneralElements/UnitFrameHud.cs

```diff
@@ -225,6 +225,11 @@ private uint BackgroundColor(Chara chara)
                 return Config.CustomBackgroundColor.Base;
             }
 
+            if (Config.UseJobColorAsBackgroundColor)
+            {
+                return ImGui.ColorConvertFloat4ToU32(GlobalColors.Instance.SafeColorForJobId(chara.ClassJob.Id).Vector);
```

Comment:
`return GlobalColors.Instance.SafeColorForJobId(chara.ClassJob.Id).Base`

Reply:
okay ill make the change here